window.lisp

```diff
@@ -908,20 +908,21 @@ needed."
     (when last-win
       (update-decoration last-win))))
 
-(defmethod focus-window (window)
-  "Make the window visible and give it keyboard focus."
+(defmethod focus-window (window &optional (raise t))
+  "Make the window visible and give it keyboard focus. If raise is t, raise the window."
   (dformat 3 "focus-window: ~s~%" window)
   (let* ((group (window-group window))
          (screen (group-screen group))
          (cw (screen-focus screen))
          (xwin (window-xwin window)))
+    (when raise
+      (raise-window window))
```

Comment:
@jorams: I've been using it since yesterday with :sloppy, and couldn't see any problems. I also checked the behavior of :click and :ignore briefly. Can you suggest a test?

@dbjergaard: do you mean making the optional argument in focus-window nil instead of t?